internal/cluster/boot.go

```diff
@@ -84,6 +84,27 @@ func dockerComposeBuild() error {
 	return nil
 }
 
+// DockerComposePull pulls down the most recent versions of the Docker images
+func DockerComposePull() error {
```

Comment:
This method could be private and be called by `cluster.Update()`. Consistently with `TearDown()`.